app/models/user.rb

```diff
@@ -931,9 +931,10 @@ def self.erase_user(id)
   # Does user have any unshown naming notifications?
   # (I'm thoroughly confused about what role the observation plays in this
   # complicated set of pages. -JPH)
+  # 2015-10-12 - JDC as currently written, observation appears to be lint
```

Comment:
I wouldn't change this file as part of the PR since these changes are entirely in comments.  I'd be OK with it as a separate tiny PR, but it would be even better if it resolved whatever the confusion is here.  I'm also not sure what you mean by "lint" in this context.